resources/schema.py

```diff
@@ -166,6 +166,11 @@ class Meta:
 
 
 class BerthNode(DjangoObjectType):
+    leases = DjangoConnectionField(
```

Comment:
when you have explicit fields on the class, it's best to delete the field form the `Meta.fields`. graphene iis not always reliable in how it handles the fields that are mentioned there and might generate it's own schema, not the same as what you specified in the field on the class.

Reply:
Good to know. I thought that, since I was adding the field, it also had to be added to `Meta.fields` 👍

Reply:
yeah, I learned this the hard way. bumped into some issues, when I had two different Nodes for the same model and graphene would use the wrong one, because it would generate its own field, overriding my explicit field on the class (where I used the right node).